automation/run-local-utests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+TOXWORKDIR="$HOME/.cache/.tox-nmstate"
+CONTAINER_IMAGE="nmstate/centos7-nmstate-utest"
+
+mkdir -p "$TOXWORKDIR"
+
+docker run \
+       --rm \
+       -ti \
+       -v $PWD:/workspace/nmstate:Z \
+       -v $TOXWORKDIR:$TOXWORKDIR:Z \
```

Comment:
This will create root-owned files in `$HOME/.cache/.tox-nmstate`, will it not? Would you be open to use podman in unprivileged mode to run the unit tests?

Also I think it would be better to integrate the feature of copying the current directory into the other run-tests file and also running this container image.

Reply:
It can be run without sudo if you set the docker group. But using podman sounds good to me.

We have piggy-back these tests on the integration test image. I’m fine to run it instead of the current runs, but I did not want to depend on you guys to agree to that, as it introduces changing existing stuff. That could have been a different proposal.